doc/lua_api.txt

```diff
@@ -2194,6 +2194,15 @@ Helper functions
 * `minetest.mkdir(path)`: returns success.
     * Creates a directory specified by `path`, creating parent directories
       if they don't exist.
+* `minetest.rmdir(path, recursive)`: returns success.
+    * Removes a directory specified by `path`.
+    * If `recursive` is set to `true`, the directory is recursively removed.
+      Otherwise, the directory will only be removed if it is empty.
+* `minetest.cpdir(source, destination, remove_source)`: returns success.
```

Comment:
I suggest adding `mvdir` which calls both `rmdir` and `cpdir`. It should be a pure lua function